src/main/java/org/takes/tk/TkCors.java

```diff
@@ -50,10 +51,16 @@
  * @author Endrigo Antonini (teamed@endrigo.com.br)
  * @version $Id$
  * @since 0.20
+ *
+ * @todo #684:15min Remove PMD.AvoidDuplicateLiterals exclude here and in
+ *  {@link RsPrettyJson}, {@link RsPrettyXml}. Note that this puzzle depends on
+ *  https://github.com/teamed/qulice/issues/760. Also remove some of PMD
+ *  excludes that are mentioned in https://github.com/yegor256/takes/issues/686
```

Comment:
@davvd Because #686 is the same issue (fix PMD.AvoidDuplicateLiterals). But it cannot be resolved without Qulice fix. What is the best approach I can use here?